src/actions/revisions.js

```diff
@@ -10,15 +13,58 @@ export function setStatusReady() {
 	};
 }
 
+export function setStatusNotReady() {
+	return {
+		type: 'REVISIONS_NOT_READY'
+	};
+}
+
 export function setStatusFetching() {
 	return {
 		type: 'REVISIONS_FETCHING'
 	};
 }
 
+export function setStatusDone() {
+	return {
+		type: 'REVISIONS_DONE'
+	};
+}
+
 export function setRevisions( revisions ) {
```

Comment:
Not in use anymore. Looks like other actions in this file are not being used as well. Please check